sys/include/net/ng_netconf.h

```diff
@@ -81,8 +81,9 @@ typedef enum {
     NETCONF_STATE_SLEEP,            /**< sleep mode */
     NETCONF_STATE_IDLE,             /**< idle mode */
     NETCONF_STATE_RX,               /**< receive mode */
-    NETCONF_STATE_TX,               /**< transmit mode */
-    NETCONF_STATE_RESET,            /**< reset mode */
+    NETCONF_STATE_TX,               /**< triggers a transmition */
+    NETCONF_STATE_RESET,            /**< triggers a hardware reset. The resulting
+                                         state of the radio device is NETCONF_STATE_IDLE */
```

Comment:
Style question: I normally do

```C
/**< triggers a hardware reset. The resulting
 *   state of the radio device is NETCONF_STATE_IDLE */
```

Since we normally do multiline comments like this.
What is preferred?